.github/workflows/python-test.yml

```
@@ -43,7 +43,7 @@ jobs:
           - os: "windows-latest"
             python-version: "3.12"
     runs-on: ${{ matrix.os }}
-    if: "!startsWith(github.event.head_commit.message, 'Version updated')"
+    if: '!startsWith(github.event.head_commit.message, "Version updated")'
```

actionlint (1.7.4)

got unexpected character '"' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '. do you mean string literals? only single quotes are available for string delimiter

(expression)